web/controllers/auth_controller.ex

```diff
@@ -1,17 +1,26 @@
 require IEx
 defmodule Janitor.AuthController do
   use Janitor.Web, :controller
-  alias Janitor.User 
+  use Timex
+  alias Janitor.User
 
   def connect(conn, _params) do
     redirect conn, external: Google.authorize_url!(scope: "email profile")
   end
 
-  def oauth(conn, %{"code" => code}) do 
+  def oauth(conn, %{"code" => code}) do
     params = token(code) |> get_user! |> map_params
     changeset = User.registration_changeset(%User{}, params)
-    User.find_or_create(changeset)    
-  end 
+    token = User.find_or_create(changeset) |> sign_jwt_token
+    conn = put_req_header(conn, "authorization", "bearer #{token}")
```

Comment:
@Xypus I think that's kind of wrong interpretation of authorization header. We won't do any authorization at this point, we just need to transport token to the caller. Why don't we just do 
```
redirect conn, to "localhost:5000?token=#{token}"  
``` 
Then we can listen on app start on SPA application.